src/components/atoms/infoCard/InfoCard.module.scss

```diff
@@ -0,0 +1,23 @@
+@import "../../../foundations/sass/all";
+
+.info-card {
+  @include center;
+  gap: map-get($spacing, sm);
+  // background-color: $primary-color;
+  color: $white-floral;
+
+  &__icon {
+    width: map-get($spacing, mds);
+    height: map-get($spacing, mds);
+
+    svg {
+      width: 100%;
+      height: 100%;
+    }
+  }
+
```

Comment:
Why do we need to set SVG width and height 100%?

Reply:
> Why do we need to set SVG width and height 100%?

i fixed the that in my own local git,